configs/nextflow.hopper.config

```diff
@@ -300,4 +315,51 @@ myeloid {
     params.gens_accessdir = "/access/myeloid_const/plot_data"
     params.accessdir = "/access/myeloid_const/"
   } 
+
+modycf {
+    params.outdir = "${params.resultsdir}${params.dev_suffix}"
+    params.subdir = 'modycf'
+    params.crondir = "${params.outdir}/cron/"
+    params.intersect_bed = "${params.refpath}/bed/twist-modycf/MODY_CF_updated.bed"
+    params.align = true
+    params.varcall = true
+    params.annotate = true
+    params.gatkcov = false
+    params.exome = false
+    params.sv = true
+    params.gatkcnv = true
+    params.str = false
+    params.onco = false
+    params.umi = true
+    params.delly = false
+    params.distributed = false
+    params.run_melt = true
+    params.run_peddy = false
+    params.prepare_gens_data = true
+    params.intervals = "${params.refpath}/bed/twist-modycf/MODY_CF_updated.interval_list"
+    params.bedgz = "${params.refpath}/bed/twist-modycf/MODY_CF_updated.bed.gz"
+    params.panelhs = "sentieon driver -r ${params.genome_file} -t ${params.cpu_all} -i "
+    params.panelhs2 = " --algo HsMetricAlgo --targets_list ${params.intervals} --baits_list ${params.intervals} hs_metrics.txt"
+    params.bed_melt = "${params.refpath}/bed/MELT/Hg38.genes.bed"
+    params.mei_list = "${params.refpath}/annotation_dbs/MELT/mei_list.txt"
+    params.meltheader = "${params.refpath}/annotation_dbs/MELT/melt_vcf_header"
+    params.svdb = "${params.refpath}/annotation_dbs/onco-solid/loqusdb/loqusdb_oncosv_export_20220318.vcf"
+    params.svrank_model_s = "/fs1/resources/scout/rank_models_38/svrank_single_v1.0_panel.ini"
+    params.rank_model = "/fs1/resources/scout/rank_models_38/rank_model_cmd_v4.ini"
+    params.rank_model_s = "/fs1/resources/scout/rank_models_38/rank_model_cmd_v4_single_withoutmodels_onco.ini"
+    params.svrank_model = "/fs1/resources/scout/rank_models_38/svrank_model_v1.0.ini"
+    params.vcfanno = "${params.refpath}/annotation_dbs/wgs/vcfanno/vcf_anno"
+    params.cnvkit_reference = "${params.refpath}/annotation_dbs/modycf/cnvkit_ref/masked/modycf_pon_cnvkit_masked.cnn"
+    params.loqusdb = "loqusdb_modycf"
+    params.antype = "panel"
+    params.assay = "modycf"
+    params.noupload = false
+    params.scoutbed = "${params.refpath}/bed/twist-modycf/modycf_all_targets_onlycoding.bed"
+    params.gatk_intervals = "${params.refpath}/annotation_dbs/modycf/gatk_ref/masked/modycf_pon.preprocessed.blacklisted.gcfiltered.noalt.interval_list"
+    params.ploidymodel = "${params.refpath}/annotation_dbs/modycf/gatk_ref/masked/modycf_pon_ploidy-model"
+    params.gatkreffolders = "${params.refpath}/annotation_dbs/modycf/gatk_ref/masked/gatk_ref_modycf_masked"
+    params.gens_accessdir = "/access/wgs/plot_data"
+    params.accessdir = "/access/modycf/"
+  }
+
```

Comment:
perhaps our profiles would benefit from having categories in the future

Reply:
Yes